packages/commons/src/events/FieldConfig.ts

```diff
@@ -470,13 +470,10 @@ export type AdministrativeArea = z.infer<typeof AdministrativeArea>
 const LocationInput = BaseField.extend({
   type: z.literal(FieldType.LOCATION),
   defaultValue: NonEmptyTextValue.optional(),
-  configuration: z
-    .object({
-      searchableResource: z
-        .array(z.enum(['locations', 'facilities', 'offices']))
-        .optional()
-    })
-    .optional()
+  configuration: z.object({
+    searchableResource: z.array(z.enum(['locations', 'facilities', 'offices'])),
+    status: z.enum(['active', 'inactive'])
```

Comment:
Maybe the status field could be optional so no status = both active and inactive shown. 
Or we can make it not configurable at all and always only show active ones in inputs. Outputs need to show all